templates/stripes/stripe-reference.php

```diff
@@ -0,0 +1,12 @@
+<?php if (!empty($referencePage)) : ?>
+<?php $reference_type_template = $template_name . '--' . $referencePage['type']; ?>
+<div class="<?= $this->e($type, 'wrapperClasses') ?>">
+  <?php
+  if ($this->engine->exists("$reference_type_template")) {
+    $this->insert($reference_type_template, get_defined_vars());
+  } else {
+    $this->insert("$template_name--default", get_defined_vars());
```

Comment:
Lines 2 & 8 should probably treat the variable + string stuff the same.

And if you're doing it this way I prefer "{$template_name}--default" (functionally the same) so it's clear (a) that there's a variable in there and (b) where the variable ends.